tex/artikel/FSR_Vorstellungsrunde.tex

```diff
@@ -24,95 +24,220 @@ \section{Der Fachschaftsrat~(FSR) Physik stellt sich vor}
 
 \fibelvorstellung{
 	\begin{wrapfigure}{l}{0cm}
-		\includegraphics[width=\fibelstdlen]{res/vorstellungsfotos/anna_niemann_cropped.PNG}
+		\includegraphics[width=\fibelstdlen]{res/vorstellungsfotos/michael_te_vrugt}
 	\end{wrapfigure}
 }
-{Hey, ich heiße Anna und komm nun in mein drittes Mastersemester. Bei Fragen rund ums
-	Studium und Münster könnt ihr mich gerne immer ansprechen.
-	Wünsche euch einen wunderschönen Start ins Studium:)
+{
+Hi, ich bin Michael und promoviere gerade in Physik und Philosophie. In der Fachschaft bin ich seit Jahren aktiv und momentan u.A. für die Master-Infoveranstaltung zuständig. 
+Bei Fragen zu Philosophie, einem Auslandsjahr, Doppelstudium oder Stipendium - und natürlich auch allem anderen - seid ihr bei mir genau richtig. 
+Ansonsten wünsche ich euch viel Spaß in der O-Woche, und vielleicht sieht man sich mal bei einer Fachschaftssitzung. :)
 }
 
+
 \fibelvorstellung{
 	\begin{wrapfigure}{r}{0cm}
-		\includegraphics[width=\fibelstdlen]{res/vorstellungsfotos/benedikt_bieringer.png}
+		\includegraphics[width=\fibelstdlen]{res/vorstellungsfotos/kristin_nissen.jpg}
 	\end{wrapfigure}
 }
-{Hallo zusammen! Mein Name ist Benedikt. In der Fachschaft beschäftige ich mich unter anderem mit Computergrafik/Design. Programmieren und Schwimmen sind nur zwei meiner weiteren Freizeitbeschäftigungen. In meinen mittlerweile schon 12~Semestern Fachschafts- und Studienerfahrung kann ich euch aber auch bei einer ganzen Reihe weiterer Fragen weiterhelfen.
+{
+Hey ihr lieben! Ich bin Kristin und nun im 1. Semester meines Physikmasters. Ich kümmere mich in der Fachschaft um die Evaluation und auch um die Homepage. 
+Falls ihr irgendwelche Fragen habt werde ich euch liebend gerne weiter helfen. Nun wünsche ich euch erstmal einen reibungsfreien Studienbeginn und viel Spaß in Münster. 
 }
 
+
 \fibelvorstellung{
 	\begin{wrapfigure}{l}{0cm}
-		\includegraphics[width=\fibelstdlen]{res/vorstellungsfotos/hauke_hawighorst.jpg}
+		\includegraphics[width=\fibelstdlen]{res/vorstellungsfotos/Fotos Selbstvorstellungstexte Fibel/Jonas L.jpg}
 	\end{wrapfigure}
 }
-{Moin, ich heiße Hauke und bin seit 2016 an der Uni und in der Fachschaft. Als Erasmus Student war ich in Sevilla (Spanien) und in Münster bin ich für die Evaluation der Lehre Zuständig. Euch ein herzliches Willkommen in Münster!
+{
+Moin zusammen! Ich bin Jonas und studiere mittlerweile im Master Physik. In meinem Bachelor habe ich außerdem ziemlich viel Mathe gemacht (und bin da auch eigentlich immer noch nicht ganz von losgekommen). 
+Zudem bin ich im Moment Vorsitzender der Fachschaft. Wenn ihr also Fragen zum Nebenfach Mathe oder zur Fachschaft an sich habt, bin ich ein guter Ansprechpartner :-)
 }
 
 
 \fibelvorstellung{
 	\begin{wrapfigure}{r}{0cm}
-		\includegraphics[width=\fibelstdlen]{res/vorstellungsfotos/michael_te_vrugt}
+		\includegraphics[width=\fibelstdlen]{res/vorstellungsfotos/Fotos Selbstvorstellungstexte Fibel/Anna N.PNG}
```

Comment:
```suggestion
		\includegraphics[width=\fibelstdlen]{res/vorstellungsfotos/Fotos Selbstvorstellungstexte Fibel/Anna N.jpg}
```

Reply:
All das Angemerkte hab ich vorab eig schon ausgebessert .. ô.ô 
Deswegen hab ich mich ja auch die ganze Zeit gewundert, dass meine Änderungen nachwievor nicht hier auftauchen :p

Reply:
@willernicht Sicher, dass wir über das gleiche reden? An dieser Stelle habe ich nur eine Sache angemerkt, und zwar, dass der Unterordner "Fotos Selbstvorstellungstexte Fibel" im Ordner "vorstellungsfotos" so wirkt, als könnte man ihn vermeiden, indem man Bilder direkt in "vorstellungsfotos" hochlädt, was das Ganze übersichtlicher machen würde.

Reply:
Kümmert euch lieber mal darum, dass ich es geschissen kriege, mir ENDLICH MAL eine SCHEIß PDF ausgeben zu lassen!!!